### DISCUSSION TITLE
Matplotlib widget in VSCode Remote Session

### DISCUSSION THREAD

**Discussion by oravus** [microsoft/vscode-jupyter#9328] [Tips, Tricks and Workarounds]
Hi all, the following might be relevant for someone trying to use matplotlib widget through VSCode's remote-ssh connection:

Include `ForwardX11 yes` in your ssh config as below:
```
Host <host>
  HostName <hostname>
  User <username>
  ForwardX11 yes
```

reconnected remote and tested as below:

```
%matplotlib widget

import matplotlib.pyplot as plt
import numpy as np

plt.imshow(np.ones([10,10]))
plt.show()
```

**Comment by greazer** [comment]
Thank you for posting this suggestion @oravus!
